contrib/prometheus/prometheus.yml

```diff
@@ -12,4 +12,4 @@ scrape_configs:
     # scheme defaults to 'http'.
 
     static_configs:
-      - targets: ["host.docker.internal:8000"]
+      - targets: ["oracle:8002"]
```

Comment:
It it ok that we are pointing to 8001 and 8002 in some places?

Reply:
the 8001 is the app one, the 8002 one is the oracle side car metrics